src/helper_functions.jl

```diff
@@ -1,6 +1,8 @@
 export sign_cadlag,
        check_method_implementation,
-       check_method_ambiguity_binary
+       check_method_ambiguity_binary,
+       @commutative_neutral,
```

Comment:
i'm not sure that we want to export these names, since this is more of an internals of the package (same comment applies to the 2 `check_...` methods of above)

Reply:
Good point.

But I think we need to export the check methods for usage in the unit tests (not tested). Somewhere I proposed that we remove them from the code base and move them to the unit tests. Any opinion?

Reply:
even if it is the name is not exported, one can use the function by writing the module's name, as in `LazySets.check_method_implementation`, or it's used in many places do a `import LazySets.check_method_implementation`.

but i agree to move those check methods to the tests folder 👍

Reply:
> one can use the function by writing the module's name

You are right, I forgot that.

> but i agree to move those check methods to the tests folder 

Good, I changed #197 accordingly.